superset/migrations/shared/migrate_viz/processors.py

```diff
@@ -31,6 +31,39 @@ def _pre_action(self) -> None:
             self.data["metric"] = self.data["metrics"][0]
 
 
+class TimeseriesChart(MigrateViz):
+    has_x_axis_control = True
+
+    def _pre_action(self) -> None:
+        contribution = self.data.get("contribution")
+        if contribution:
+            self.data["contributionMode"] = "row" if contribution else None
+
+        show_brush = self.data.get("show_brush")
+        self.data["zoomable"] = False if show_brush == "no" else True
+
+        self.data["markerEnabled"] = self.data.get("show_markers")
+        self.data["y_axis_showminmax"] = True
+
+        x_axis_label = self.data.get("x_axis_label")
+        bottom_margin = self.data.get("bottom_margin")
+        if x_axis_label and (not bottom_margin or bottom_margin == "auto"):
+            self.data["bottom_margin"] = 30
+
+        rolling_type = self.data.get("rolling_type")
+        if rolling_type:
+            self.data["rolling_type"] = None if rolling_type == "None" else rolling_type
```

Comment:
I presume that if the value isn't explicitly defined it defaults to `None` so maybe,

```python
if (rolling_type := self.data.get("rolling_type")) and rolling_type != "None":
     self.data["rolling_type"] = rolling_type
```